http/headers/feature-policy.json

```diff
@@ -576,48 +495,22 @@
               "edge": {
                 "version_added": "79"
               },
-              "firefox": [
-                {
-                  "version_added": "74",
-                  "partial_implementation": true,
-                  "notes": [
-                    "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements.",
-                    "Before Firefox 80, applying <code>fullscreen</code> to an <code>&lt;iframe&gt;</code> (i.e. via the <code>allow</code> attribute) does not work unless the <code>allowfullscreen</code> attribute is also present."
-                  ]
-                },
-                {
-                  "version_added": "65",
-                  "notes": "Before Firefox 80, applying <code>fullscreen</code> to an <code>&lt;iframe&gt;</code> (i.e. via the <code>allow</code> attribute) does not work unless the <code>allowfullscreen</code> attribute is also present.",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "dom.security.featurePolicy.header.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
-              "firefox_android": [
-                {
-                  "version_added": "79",
-                  "partial_implementation": true,
-                  "notes": [
-                    "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements.",
-                    "Before Firefox 80, applying <code>fullscreen</code> to an <code>&lt;iframe&gt;</code> (i.e. via the <code>allow</code> attribute) does not work unless the <code>allowfullscreen</code> attribute is also present."
-                  ]
-                },
-                {
-                  "version_added": "65",
-                  "notes": "Before Firefox 80, applying <code>fullscreen</code> to an <code>&lt;iframe&gt;</code> (i.e. via the <code>allow</code> attribute) does not work unless the <code>allowfullscreen</code> attribute is also present.",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "dom.security.featurePolicy.header.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                }
-              ],
+              "firefox": {
+                "version_added": "74",
+                "partial_implementation": true,
+                "notes": [
+                  "Only supported through the <code>allow</code> attribute on <code>&lt;iframe&gt;</code> elements.",
+                  "Before Firefox 80, applying <code>fullscreen</code> to an <code>&lt;iframe&gt;</code> (i.e. via the <code>allow</code> attribute) does not work unless the <code>allowfullscreen</code> attribute is also present."
```

Comment:
The behavior described here sounds indistinguishable from no support before Firefox 80, assuming just the `allowfullscreen` attribute is enough and adding the `allow` attribute doesn't change anything. Can you check where this note came from and if it makes any sense?

Maybe split this out into a separate PR if it needs tweaking beyond the script-made changes.

Reply:
It appears the notes came from #6493, as well as #12548.

Reply:
OK, maybe this can be simplified then.

Reply:
I opened #16537 for us to take a look into this at a later time!